install/defconf/fossology.conf.in

```diff
@@ -77,6 +77,17 @@ PROJECTSTATEDIR=$LOCALSTATEDIR/lib/$PROJECT
 ; cache folder
 CACHEDIR=$LOCALSTATEDIR/cache/$PROJECT
 
+; Use external Authentication
+CONF_EXT_AUTH_ENABLE=false
+; Environment variable where to find the User name
+CONF_EXT_AUTH_ENV_USER=
+; Environment variable where to find the User email
+CONF_EXT_AUTH_ENV_MAIL=
+; Environment variable where to find the User Description
+CONF_EXT_AUTH_ENV_DESC=
+; Set to true to use lowercase usernames
+CONF_EXT_AUTH_ENV_LOWERCASE_USER=
+
```

Comment:
It would be a better idea to move these configurations under their own section (something like `EXT_AUTH`).
Then with the help of https://github.com/fossology/fossology/blob/8a17b9532f3b729451a126481555a59af5facaa5/src/lib/php/common-sysconfig.php#L73 , you can access variables using following syntax:

`$GLOBALS['SysConf']['EXT_AUTH']['CONF_EXT_AUTH_ENABLE']`.

**Note:** The function is called here: https://github.com/fossology/fossology/blob/e3263098abee94acac1bf9bc91903ecbf51c1a8b/src/www/ui/index.php#L52